ChangeLog

```diff
@@ -9,6 +9,12 @@ Features
      Contributed by Jack Lloyd and Fortanix Inc.
    * Add the Wi-SUN Field Area Network (FAN) device extended key usage.
    * Add the oid certificate policy x509 extension.
+   * Add support for parsing otherName entries in the Subject Alternative Name
```

Comment:
Blocker (sorry, should have said it earlier): Given that we had community requests on the extended parsing of SAN's, we should explicitly state here that we now include all types of SAN's in the corresponding field in the CRT. Also, we should list all GitHub issues that this PR fixes.

Reply:
The only issue that this PR addressees, but not fully fixes , is #459, IMHO

We don't parse the SAN items to a readable \ known format, we only add them to the subject_alt_name member, and a user can parse their own.

Reply:
Yes, but that's already a big gain for users, because they can continue from there.

Reply:
Anyway, I believe I have addressed this in e269537b807ce066cc31fe10f5c8d975f19c8362